packit_service/models.py

```diff
@@ -840,3 +836,9 @@ def create(cls, event):
                 ]
                 session.add(installation)
             return installation
+
+    def __repr__(self):
+        return f"InstallationModel(id={self.id}, account={self.account_login})"
+
+    def __str__(self):
+        return self.__repr__()
```

Comment:
`__str__` is I think not needed if it's the same `__repr__`.
From [docs](https://docs.python.org/3/reference/datamodel.html#object.__repr__):
"If a class defines `__repr__()` but not `__str__()`, then `__repr__()` is also used when an “informal” string representation of instances of that class is required."

Reply:
It is used for every class in models, so can I remove it everywhere?

Reply:
@jpopelka you're idd correct - `__repr__` is better and we don't need `__str__`:
```python
class C:
    a = 1
    b = "2"

    def __str__(self):
        return f"C(a={self.a}, b={self.b})"


class D:
    a = 1
    b = "2"

    def __repr__(self):
        return f"D(a={self.a}, b={self.b})"


print(f"str(C()): {str(C())}")
print(f"str(C()): {C()}")
print(f"str(D()): {str(D())}")
print(f"str(D()): {D()}")
print(f"repr(C()): {repr(C())}")
print(f"repr(D()): {repr(D())}")
```

```
str(C()): C(a=1, b=2)
str(C()): C(a=1, b=2)
str(D()): D(a=1, b=2)
str(D()): D(a=1, b=2)
repr(C()): <__main__.C object at 0x7fae2cce9fd0>
repr(D()): D(a=1, b=2)
```